lib/kafka/client.rb

```diff
@@ -216,8 +216,10 @@ def async_producer(delivery_interval: 0, delivery_threshold: 0, max_queue_size:
     #   not triggered by message processing.
     # @param heartbeat_interval [Integer] the interval between heartbeats; must be less
     #   than the session window.
+    # @param offset_retention_time [Integer] the time period that committed
+    #   offsets will be retained, in seconds. If -1, then the broker default is used.
```

Comment:
Yes, this is the value that the broker expects, and `ruby-kafka` uses this in the protocol class: https://github.com/zendesk/ruby-kafka/blob/master/lib/kafka/protocol/offset_commit_request.rb#L5